### DISCUSSION TITLE
Editing and testing webhook is not working [Jun 08]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2385] [Discord Threads]
> 🙋‍♂️ **alex.pro**

**Comment by o1lab** [comment]
> 🙋‍♂️ **alex.pro**  
#### Node: **v12.22.12**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### Database: **mysql2**   
#### ProjectOnRootDB: **true**   
#### RootDB: **mysql2**   
#### PackageVersion: **0.91.7**  
#  
> 💁 **o1lab**  
#### can u hit save before you hit test  
#  
> 🙋‍♂️ **alex.pro**  
#### the same... you can see it works fine with first webhook, but doesn't work with others  
#  
> 🙋‍♂️ **alex.pro**  
#### <@310157751395287040>  
#  
> 💁 **o1lab**  
#### yep, we will look into it 🙂  
#  
> 💁 **o1lab**  
#### https://github.com/nocodb/nocodb/issues/2277  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/983285304599404544)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
